Source/PrivateHeaders/Ably/ARTTypes+Private.h

```diff
@@ -15,12 +15,18 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nonatomic, readonly, nullable) ARTRetryAttempt *retryAttempt;
 
+/**
+ * Indicates whether the connection was resumed.
+ */
+@property (assign, nonatomic) BOOL resumed;
```

Comment:
no need for `resumed`

Reply:
Yeah, you can remove it as per https://github.com/ably/ably-cocoa/pull/1714/files#r1451244126

Reply:
Duplicate of https://github.com/ably/ably-cocoa/pull/1714#discussion_r1423854723